src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,77 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.17.0.dev0 (5/13/2019)
+-----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* V2 `./pants test.pytest` selects interpreter based off of compatibility constraints (#7679)
+  `PR #7679 <https://github.com/pantsbuild/pants/pull/7679>`_
+
+Bugfixes
+~~~~~~~~
+
+* Mark build-support Python files as Pants targets to lint build-support (#7633)
+  `PR #7633 <https://github.com/pantsbuild/pants/pull/7633>`_
+
+* Fix dependency cycle with cache folder (#7693)
+  `PR #7693 <https://github.com/pantsbuild/pants/pull/7693>`_
+
+* Add pantsd_invalidation_globs option into pants.ini (#7705)
+  `PR #7705 <https://github.com/pantsbuild/pants/pull/7705>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Refactor `check_banned_imports.py` to allow Python 3 files to import `subprocess` (#7706)
+  `PR #7706 <https://github.com/pantsbuild/pants/pull/7706>`_
+
+* Rerun cbindgen on changes to src (#7708)
+  `PR #7708 <https://github.com/pantsbuild/pants/pull/7708>`_
+
+* add stack traces to unhashable TypeErrors in the engine (#7532)
+  `PR #7532 <https://github.com/pantsbuild/pants/pull/7532>`_
+
+*  Convert `check_banned_imports.sh` to Python to workaround Bash array issues and for less duplication (#7702)
+  `PR #7702 <https://github.com/pantsbuild/pants/pull/7702>`_
+>`_ttps://github.com/pantsbuild/pants/pull/7698#discussion_r282947998.
+  `PR #6747 <https://github.com/pantsbuild/pants/pull/6747>`_
+
+* Output stderr in V2 test rule (#7694)
```

Comment:
@stuhood same thing - should this be a feature or be bugfix or be tooling? It's adding functionality that V1 already had.

cc @illicitonion @cosmicexplorer

Reply:
Fine with either.

Reply:
@ity, Because we don't anticipate anyone using V2 in the real world (it's almost ready, but not quite), I recommend moving this and all V2 enhancement-related code to tooling / refactors.

When reading release notes, I suspect users spend more time on bugfixes and new features, and less time on tooling. I'd like to make it easier for them to know that they can skip this PR / it has little impact on them.